data/mods/Xedra_Evolved/damage_types.json

```diff
@@ -61,5 +61,55 @@
     "pet_prot_info": { "order": 999999, "show_type": false },
     "melee_combat_info": { "order": 999999, "show_type": false },
     "ablative_info": { "order": 999999, "show_type": false }
+  },
+  {
+    "id": "xe_shadow_damage",
+    "type": "damage_type",
+    "physical": false,
+    "magic_color": "dark_gray",
+    "name": "sharp edged shadows",
+    "skill": "cutting",
+    "derived_from": [ "cut", 0.8 ],
+    "immune_flags": { "character": [ "SHADOW_CHAR_IMMUNE" ], "monster": [ "SHADOW_MONSTER_IMMUNE" ] },
+    "ondamage_eocs": [ "EOC_SHADOW_DAMAGE_BLIND_CHANCE", "EOC_SHADOW_DAMAGE_EXTRA_BLIND_CHANCE" ]
+  },
```

Comment:
While not mundane, these kind of attacks appear to be in part motion-based (not a physical attack, but nevertheless "transported" by mundane motion, so to say). So like Momentum Alteration (MoM) or the %-reductions for some paracleisans should perhaps apply here. I don't know if `ARMOR_CUT` takes  "xe_shadow_damage" into account, might want to check that.

Likewise-ish, maybe photokinetic damage (MoM) should be supereffective against shadows? (Or immune to photokinetic damage, dunno.)

And maybe Lucent Barrier (MoM) should protect against shadows, but this is a stretch, since AFAIK the light that psions control is entirely mundane.

On a similar note, maybe shadows (but not shadowbeasts?) should be immune from biological damage (don't know if that's the case currently).

Reply:
I don't know if ARMOR_CUT takes "xe_shadow_damage" into account, might want to check that.  

The derived from stat means that you will automatically get 80% of your cut armor against shadow damage.  If you are referring to enchantments that I don't have a clue on. @dseguin  When you jsonized damage do enchantments apply their armor before, after  or not at all to the derived stat?

Reply:
"ARMOR_CUT" isn't a piece of armor but an enhancement, hence why I'm wondering if "derived-from" works for this.

Going by the hardcoded case-analysis in <https://github.com/CleverRaven/Cataclysm-DDA/blob/c57cd283edc6a393ee01fd737e178f0ecced49fb/src/monster.cpp#L238>, it doesn't work for this -- i.e., ARMOR_CUT doesn't apply to shadow damage.

Reply:
It doesn't make a difference, but I should have referred to the character code <https://github.com/CleverRaven/Cataclysm-DDA/blob/c57cd283edc6a393ee01fd737e178f0ecced49fb/src/character_attire.cpp#L1866> instead.

Reply:
> While not mundane, these kind of attacks appear to be in part motion-based (not a physical attack, but nevertheless "transported" by mundane motion, so to say). So like Momentum Alteration (MoM) or the %-reductions for some paracleisans should perhaps apply here. I don't know if `ARMOR_CUT` takes "xe_shadow_damage" into account, might want to check that.
> 
> Likewise-ish, maybe photokinetic damage (MoM) should be supereffective against shadows? (Or immune to photokinetic damage, dunno.)
> 
> And maybe Lucent Barrier (MoM) should protect against shadows, but this is a stretch, since AFAIK the light that psions control is entirely mundane.
> 
> On a similar note, maybe shadows (but not shadowbeasts?) should be immune from biological damage (don't know if that's the case currently).

There isn't a good way for cross mod damage types to impact each other.  Unless both mods decide to use the same damage types and then move those types to vanilla to prevent weird overwrites.